src/clj/game/cards/operations.clj

```diff
@@ -549,7 +549,7 @@
   {:on-play
    {:prompt "Choose a faceup card"
     :choices {:card #(or (and (corp? %)
-                              (rezzed? %))
+                              (faceup? %))
```

Comment:
```suggestion
                               (installed? %)
                               (faceup? %))
```
Otherwise you can dedi cards in archives lol

Reply:
Looking at the wording on a few cards (Dedi, Mumbad Construction Co.), it seems "faceup" implies "installed", unless otherwise specified.